shell/platform/tizen/channels/platform_channel.cc

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+#include <app.h>
```

Comment:
Please move this line. Otherwise looks good.
```cpp
#include "platform_channel.h"

#include <app.h>

#include ...
```

Reply:
ok